connectors/sources/mysql.py

```diff
@@ -167,7 +174,7 @@ async def validate_config(self):
         Raises:
             Exception: Configured keys can't be empty
         """
-        connection_fields = ["host", "port", "user", "password", "database"]
+        connection_fields = ["host", "port", "user", "password", "database", "tables"]
         empty_connection_fields = []
         for field in connection_fields:
             if self.configuration[field] == "":
```

Comment:
@timgrein Minor nit to valid tables condition need to be update with `if self.configuration[field] in ["", [""]]`